### DISCUSSION TITLE
如何设置输出top10的预测结果

### DISCUSSION THREAD

**Discussion by XueKX** [PaddlePaddle/PaddleOCR#13458] [Q&A]
![image](https://github.com/user-attachments/assets/63b73e80-0407-4349-ae72-586f4ad452fe)
这里的输出只有top1

**Comment by UserWangZz** [comment]
没有对应的接口函数，需要你自己实现该功能

**Reply by XueKX** [reply]
是的，我定位到了代码的位置，但是我不知道怎么修改以输出top10 ，self.input_tensor和self.output_tensors两个参数是从模型文件中获取的。是否可以给我一些建议以便我实现我的需求，谢谢
![image](https://github.com/user-attachments/assets/fb9ce623-4eaa-41dd-9cf1-ba043842e779)

**Comment by XueKX** [comment]
已实现
